test/dbus.c

```diff
@@ -16,6 +16,8 @@ struct signal_closed {
         GDBusConnection *conn;
 };
 
+char msg[50];
```

Comment:
Why is this all the way up here and not a local variable?

Reply:
Because ASSERT of greatest exits the function and after that, the message will get used. I'd prefer a local variable, bit it wouldn't be valid anymore.

Reply:
Maybe make it a static local then?

Reply:
Done and updated.